sc/social/like/tests/test_canonicalurl_view.py

```diff
@@ -72,3 +73,33 @@ def test_update_canonical_url(self):
         # canonical URL unchaged
         self.assertEqual(
             self.portal['baz'].canonical_url, 'https://example.org/plone/baz')
+
+
+class UtilPathToVirtualPathTestCase(unittest.TestCase):
+
+    layer = INTEGRATION_TESTING
+
+    def setUp(self):
+        app = self.layer['app']
+        portal = self.layer['portal']
+        self.request = self.layer['request']
+
+        if 'virtual_hosting' not in app.objectIds():
+            manage_addVirtualHostMonster(app, 'virtual_hosting')
+
+        with api.env.adopt_roles(['Manager']):
+            self.obj = api.content.create(portal, type='News Item', id='foo')
+            api.content.transition(self.obj, 'publish')
+
+    def test_get_path_to_virtual_path(self):
```

Comment:
No exactly. These tests are not about `canonical_url`s . They're mean to test getting the paths based on the request, and how it changes with VHM.